content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -277,7 +277,8 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
 ## Arbitrary Controllers and Selectors
 
 You can skip this section if you only use PDBs with the built-in
-application controllers (Deployment, ReplicationController, ReplicaSet, and StatefulSet),
+application controllers (Deployment, ReplicationController, ReplicaSet, and StatefulSet)
+or controllers that implement a [scale subresource](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource),
```

Comment:
not sure if we should have the link 2nd time since it is linked already in the beginning of this article